src/popup/views/not-authed.tsx

```diff
@@ -1,5 +1,16 @@
 import * as React from 'react'
 
 export function NotAuthed({ signInWithTwitter }: Pick<Dispatchers<UserAction>, 'signInWithTwitter'>): JSX.Element {
-  return <button onClick={signInWithTwitter}>Sign in with twitter</button>
+  return (
+    <div className="not-authed">
+      <img src="/img/roar_128.png" />
+      <button onClick={signInWithTwitter}>
+        <img src="/img/twitter_logo_blue.svg" />
+        Log in with Twitter
+      </button>
+      <p>
+        Built with ❤️ by the team at <a href="<button onClick={signInWithTwitter}>Log in with Twitter</button>">https://morehumaninternet.org</a>
```

Comment:
This link is currently nonfunctional, let's fix that. 

To start, we need `target="_blank"` which will open a webpage in a new window. The href should also be https://morehumaninternet.org. Lastly, we can omit the protocol from the displayed text. In all that will look like this

```typescript
<p>
  Built with ❤️ by the team at <a target="_blank" href="https://morehumaninternet.org">morehumaninternet.org</a>
</p>
```

Reply:
error corrected, please review. Thanks!

Reply:
made adjustment as instructed